microcluster/app.go

```diff
@@ -71,6 +71,9 @@ func App(ctx context.Context, args Args) (*MicroCluster, error) {
 
 // Start starts up a brand new MicroCluster daemon. Only the local control socket will be available at this stage, no
 // database exists yet. Any api or schema extensions can be applied here.
+// - `schemaExtensions` is a list of schema updates in the order that they should be applied.
+// - `apiEndpoints` is a list of endpoints to be served over `/1.0`.
+// - `hooks` are a set of functions that trigger at certain points during cluster communication.
 func (m *MicroCluster) Start(apiEndpoints []rest.Endpoint, schemaExtensions []schema.Update, hooks *config.Hooks) error {
```

Comment:
Apparently `const dotGoTemplate = "package %s\n\n" +` is unused?

Reply:
Yes, I missed removing the constant. thanks.